Tests/TuistGeneratorTests/ProjectMappers/SynthesizedResourceInterfaceProjectMapperTests.swift

```diff
@@ -237,3 +241,23 @@ final class SynthesizedResourceInterfaceProjectMapperTests: TuistUnitTestCase {
         )
     }
 }
+
+extension SideEffectDescriptor {
```

Comment:
is this used anywhere?

Reply:
Oh no it was a leftover I used to debug files in tests, removing it now